group_vars/all/all.yml

```diff
@@ -38,8 +38,9 @@ helm_arch: amd64
 
 # SOLR
 # Based on https://hub.docker.com/_/solr/
-solr_image: "ckan/ckan-solr-dev"
-solr_image_tag: "2.9-py2"
+solr_image: "ckan/ckan-solr"
```

Comment:
Does the new image support older version `2.9-py2` still used by multiple old projects? If not you need to fix `solr_image: "ckan/ckan-solr-dev"` for projects using `2.9-py2`

Reply:
Not sure I understand. Should I fix it to 2.10 in other projects inventory settings? Or should I fix it to 2.9-py2 so that they don't break?

Reply:
https://github.com/fjelltopp/fjelltopp-infrastructure/pull/231